ci/scripts/c_glib_test.sh

```diff
@@ -17,25 +17,29 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set -e
+set -ex
 
-source $TRAVIS_BUILD_DIR/ci/travis_env_common.sh
+source_dir=${1}/c_glib
+build_dir=${2:-${source_dir}/build}
 
-pushd $CPP_BUILD_DIR
+export LD_LIBRARY_PATH=${ARROW_HOME}/lib:${LD_LIBRARY_PATH}
+export PKG_CONFIG_PATH=${ARROW_HOME}/lib/pkgconfig
+export GI_TYPELIB_PATH=${ARROW_HOME}/lib/girepository-1.0
+export PKG_CONFIG=/usr/bin/pkg-config
 
-if [ $TRAVIS_OS_NAME == "osx" ]; then
-  # TODO: This does not seem to terminate
-  CTEST_ARGS="-E arrow-flight-test"
-fi
+pushd ${source_dir}
 
-PATH=$ARROW_BUILD_TYPE:$PATH ctest -j2 --output-on-failure -L unittest ${CTEST_ARGS}
+ruby test/run-test.rb
 
-popd
-
-# Capture C++ coverage info (we wipe the build dir in travis_script_python.sh)
-if [ "$ARROW_TRAVIS_COVERAGE" == "1" ]; then
-    pushd $TRAVIS_BUILD_DIR
-    lcov --directory . --capture --no-external --output-file $ARROW_CPP_COVERAGE_FILE \
-        2>&1 | grep -v "ignoring data for external file"
+if [[ "$(uname -s)" == "Linux" ]]; then
+    # TODO(kszucs): on osx it fails to load 'lgi.corelgilua51' despite that lgi
```

Comment:
cc @kou

Reply:
Do error logs still exist?

Anyway, we can disable this on macOS. Because this isn't important.